documentation/experimental_features.md

```diff
@@ -8,6 +8,30 @@ API might change, requiring the user to update their code or configuration. The
 Bolt team attempts to make these changes painless by providing useful warnings
 around breaking behavior where possible. 
 
+## Configuration policies and `bolt policy` subcommand
+
+This feature was introduced in [Bolt 3.21.0](). 
+
+Configuration policies are public classes that can be assigned directly to one
```

Comment:
```suggestion
Configuration policies, or policies for short, are public classes that can be applied directly to one
```

Reply:
Assigning policies to nodes in a way that persists beyond one application requires what we call a "classifier" or external node classifier, which is only going to exist in Cygnus, not Bolt.